src/SearchResults.js

```diff
@@ -0,0 +1,51 @@
+import moment from "moment";
+import React, { useState } from "react";
+
+const SearchResults = props => {
+  const [color] = useState("");
```

Comment:
On line 5, you have not declared the `set` method for the state and you never call it. Because of this you are not using the power of state to trigger the re-render of your component for you automatically.

I think you can make a change to the way you have implemented the row highlighting to use state better.

My suggestion, create a new component `SearchResultRow` that contains lines 31- 43, and replace those lines with your new component
```
<tbody className="text-center">
          {props.results.map((person, index) => (
            <SearchResultRow key={index} person={person} />
          ))}
</tbody>
```

Move both your `state` and `onClickHandler` into this new component. Now, for each instance of your SearchResultRow (aka for each row), it will have it's own state keeping track of whether it is selected or not.

Now, whenever `onClick` is called, all your function has to do is `set` the status to indicate if the row is highlighted or now. When the `set` method is called, it will automatically trigger a re-render.

You can then use the state to change the `className` using a ternary operator. Something like
```
<tr onClick={highlightRow} className={isSelected ? "gray" : ""}>
```